From 665c82bb1f33396091fa451b4032da333ee8e386 Mon Sep 17 00:00:00 2001
From: Ofer Heifetz <oferh@marvell.com>
Date: Sun, 29 Oct 2017 15:41:36 +0200
Subject: [PATCH 2039/2241] tools: nas: enable skip_copy on raid topologies

Enable kernel built-in skip_copy on md devices, improves write
performance.

Change-Id: I51cb274b8b3927699039a6c5347c4e298bffe03e
Signed-off-by: Ofer Heifetz <oferh@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/45654
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 tools/nas/nas_init.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/nas/nas_init.sh b/tools/nas/nas_init.sh
index c3af31a..98555e6 100755
--- a/tools/nas/nas_init.sh
+++ b/tools/nas/nas_init.sh
@@ -614,6 +614,7 @@ else # RAID TOPOLOGY
 	echo 100000 > /sys/block/md0/md/sync_speed_min
 
 	echo $CPU_COUNT > /sys/block/md0/md/group_thread_cnt
+	echo          1 > /sys/block/md0/md/skip_copy
 
 	set +o verbose
 	echo -ne "[Done]\n"
-- 
2.7.4

